src/components/dependencies/RescoringModal.js

```diff
@@ -427,6 +427,15 @@ const RescoringFilterOption = ({
   const theme = useTheme()
   const [selected, setSelected] = React.useState([])
 
+  React.useEffect(() => {
+    // remote selected filter if option is not available anymore (e.g. last rescoring was deleted)
+    if (selected.every((s) => options.some((o) => o.name === s))) return
+    // ensure to only update state if required
```

Comment:
I think the placement of this comment is somewhat misleading, as it refers to the line above, does it not?